lib/ansible/modules/net_tools/ipwcli_dns.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Christian Wollinger <cwollinger@web.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: ipwcli_dns
+
+short_description: Manage DNS Records for Ericsson IPWorks via ipwcli
+
+version_added: "2.10"
+
+description:
+    - "Manage DNS records for the Ericsson IPWorks DNS server. The module will use the ipwcli to deploy the DNS records."
```

Comment:
I might have missed it, but I do not see the requirement for the CLI utility ipwcli mentioned in the module documentation; that ought to be added.

Reply:
The ipwcli is pre-installed on the IPWorks system. It is not a Python packet. Should I add it to the docs?

Reply:
I think it's best unless it's clear from the module documentation that this module must be run on a system which has that installed.